bftengine/src/bftengine/ControllerWithSimpleHistory.hpp

```diff
@@ -33,17 +37,35 @@ class ControllerWithSimpleHistory : public ControllerBase {
 
   virtual CommitPath getCurrentFirstPath() override;
   virtual uint32_t timeToStartSlowPathMilli() override;
+  // Timer that determines the frequency of checking whether to downgrade active requests to slow path
   virtual uint32_t slowPathsTimerMilli() override;
 
   // events
 
+  // Initialize controller, in case replica is Primary in new view
   virtual void onNewView(ViewNum v, SeqNum s) override;
+
+  // Called after executing all transactions in a block -
```

Comment:
as pointed out by Andrew, let's use ":" at the end of the line, when enumerating sub-items.

Reply:
@teoparvanov done